iFixit/react-commerce
# Fix filtered price range label bug

This PR fixes #45 

## QA

1. Visit the Vercel PR preview
2. Visit the parts collection page
3. Open price range filter and select `$10 - $25` option
4. Verify that the filter `$10 - $25` option text **does not** change to `Under $25`
